src/schrodinger/rdkit_extensions/helm/cg_coordgen.cpp

```diff
@@ -694,49 +766,10 @@ unsigned int compute_monomer_mol_coords(RDKit::ROMol& monomer_mol)
     // SHARED-9795: Special case for single polymers that are strictly chains
     if (is_single_linear_polymer(monomer_mol)) {
         lay_out_snaked_linear_polymer(*polymers[0]);
-    } else {
-        for (size_t i = 0; i < polymers.size(); i++) {
-            auto polymer = polymers[i];
-            if (i == 0) {
-                lay_out_polymer(*polymer);
-                continue;
-            }
-
-            auto last_polymer = polymers[i - 1];
-            const auto polymer_bonds =
-                get_bonds_between_polymers(*polymer, *last_polymer);
-            auto rotate_polymer =
-                !polymer_bonds.empty() &&
-                (polymer_bonds.size() > 1 || is_nucleic_acid(*polymer));
```

Comment:
I have no idea why I rotated the second polymer to begin with. What you have in your description definitely seems better to me.